java/src/main/java/ai/rapids/cudf/PinnedMemoryPool.java

```diff
@@ -226,7 +227,12 @@ public static long getAvailableBytes() {
     return 0;
   }
 
-  private PinnedMemoryPool(long poolSize) {
+  private PinnedMemoryPool(long poolSize, int gpuId) {
```

Comment:
I'd prefer to see two constructors here, the one that used to exist which calls this new API with `-1` for the GPU ID.  Then callers don't have to figure out what to pass for a gpuId when they don't have one.  That also eliminates the need to adjust all the current call sites and makes the change much smaller.

Reply:
Added different initialize function that doesn't have the parameter since that is what htis is being called from and was able to remove the test changes